fluxcd/federated-etl-flux/plugins/README.md

```diff
@@ -0,0 +1,83 @@
+# Kubecost Additional Plugins
```

Comment:
```suggestion
# Kubecost Actions
```

Carbon costs is just an optional feature. everything else is considered Kubecost actions.